### DISCUSSION TITLE
Pass arguments from Turbo to Yarn scripts

### DISCUSSION THREAD

**Discussion by openscript** [vercel/turborepo#5679] [Help] [answered]
If I run something like `turbo run lint -- --fix` I get the following warning:

`warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts`

How to workaround this?

**Comment by anthonyshew** [comment, marked as answer]
Closing here for https://github.com/vercel/turbo/issues/7864
